lightning/common/security.go

```diff
@@ -146,7 +147,11 @@ func (tc *TLS) WrapListener(l net.Listener) net.Listener {
 
 // GetJSON obtains JSON result with the HTTP GET method.
 func (tc *TLS) GetJSON(path string, v interface{}) error {
-	return GetJSON(tc.client, tc.url+path, v)
+	return tc.GetJSONWithContext(context.TODO(), path, v)
```

Comment:
```suggestion
	return tc.GetJSONWithContext(context.Background(), path, v)
```

i don't think you could use other context here, so no TODO. (or just remove the entire non-Context function)